src/ZoneServer/World/Actors/Monsters/Mob.cs

```diff
@@ -282,6 +282,15 @@ public bool TakeDamage(float damage, ICombatEntity attacker)
 			if (this.Components.TryGet<AiComponent>(out var ai))
 				ai.Script.QueueEventAlert(new HitEventAlert(this, attacker, damage));
 
+			if (attacker is Character attackCharacter)
```

Comment:
Frankly, this doesn't feel like it belongs here. It _works_, the follower of the attacker gets informed about the attack, but, again, don't you think the follower should be able to react on its own?

This pattern has a high risk of quickly getting unwieldy, because the moment we add pets we'd have to notify those too. Then we might want to add something like a clone command like in RO, which would add another type of follower. Afaik there's also skills that summon followers, and so on. If we were to move the handling to the AI script though they could all just do their thing, independently of the Character and Mob classes.

Reply:
Yeah, I had a similar thought process about it, though under the current implementation it seems to be the only way it can be done.  As I mentioned above, if anyone nearby was informed whenever something gets hit, neither of those checks would be necessary, but it's a fair bit of a change to the way the hit events are sent.

Reply:
Well, as long as we're in agreement that it should be updated soon^^